ticdc/ticdc-sink-to-kafka.md

```diff
@@ -531,4 +531,21 @@ If the message contains the `claimCheckLocation` field, the Kafka consumer reads
 }
 ```
 
```

Comment:
These changes are not needed in this PR, right? They are already included in https://github.com/pingcap/docs/pull/18903/files.